### DISCUSSION TITLE
How to use different logos for the login and the sidebar ?

### DISCUSSION THREAD

**Discussion by gausoft** [filamentphp/filament#13976] [Help] [answered]
### Package

Other

### Package Version

v3.0.0

### How can we help you?

When I set brandLogo in config it applies the same logo. I'd like to use two versions of the brand logo with differents size : one for the login page and one for the sidebar. Any idea? Thanks

**Comment by arcanoix** [comment]
Resolve it?

**Reply by leandrocfe** [reply, marked as answer]
```php
$panel
...
->brandLogo(fn () => auth()->check() ? asset('images/logoA.png') : asset('images/logoB.png'))
```

You can also use brandLogoHeight with the same approach
https://filamentphp.com/docs/3.x/panels/themes#adding-a-logo

**Reply by arcanoix** [reply]
i work, thanks.

**Reply by leandrocfe** [reply]
@gausoft ?

**Reply by gausoft** [reply]
Yeah. Thanks.
